trp_use_case/view/use_case_collection.xml

```diff
@@ -1,137 +1,123 @@
-                <tree editable="top">
-                    <field name="name"/>
-                </tree>
-            </field>
-        </record>
-        <record id="use_case_version_tag_tree" model="ir.ui.view">
-            <field name="name">Use case version tag tree</field>
-            <field name="model">use_case.version_tag</field>
-            <field name="arch" type="xml">
-                <tree editable="top">
-                    <field name="user"/>
-                    <field name="date"/>
-                    <field name="name"/>
-                </tree>
-            </field>
-        </record>
-        <record id="action_use_case_collections"
-                model="ir.actions.act_window">
-            <field name="name">Set of use cases</field>
-            <field name="res_model">use_case.collection</field>
-            <field name="view_type">form</field>
-            <field name="view_mode">tree,form</field>
-        </record>
-        <record id="action_use_cases"
-                model="ir.actions.act_window">
-            <field name="name">Use cases</field>
-            <field name="res_model">use_case</field>
-            <field name="view_type">form</field>
-            <field name="view_mode">tree,form</field>
-            <field name="view_id" ref="use_case_tree" />
-            <field name="context">{'active_test': False}</field>
-        </record>
-        <menuitem id="menu_use_case_main_menu"
-                  name="Use cases" />
-        <menuitem id="menu_use_case_main"
-                  name="Use cases"
-                  parent="menu_use_case_main_menu"
-                  />
-        <menuitem id="menu_use_case_collections"
-                  name="Sets of use cases"
-                  sequence="1"
-                  parent="menu_use_case_main"
-                  action="action_use_case_collections"
-                  />
-        <menuitem id="menu_use_cases"
-                  name="Use cases"
-                  parent="menu_use_case_main"
-                  action="action_use_cases"
-                  />
-
-        <act_window
-            domain="[('collection_id', '=', active_id)]"
-            id="action_use_case_collection_open_use_cases"
-            name="Use cases"
-            res_model="use_case"
-            src_model="use_case.collection"
-            view_mode="tree,form"
-            context="{
-                     'active_test': False,
-                     'default_collection_id': active_id,
-                     }"
-            view_type="form"/>
-
-    </data>
+  <data>
+    <record id="use_case_collection_form" model="ir.ui.view">
+      <field name="name">Use case collection form</field>
+      <field name="model">use_case.collection</field>
+      <field name="arch" type="xml">
+        <form>
+            <header>
+                <button string="Set to draft" type="object" name="draft_statusbar"  groups="base.group_user" attrs="{'invisible': [('state', '=', 'draft')]}"/>   
```

Comment:
I am missing a functional requirement here. Why highlight more buttons and sort them?